DemoApp/src/Console.cc

```diff
@@ -1,8 +1,10 @@
 #include <sstream>
 #ifdef _WIN32
 #include <windows.h>
-#elif defined __unix__
+#elif defined(__unix__) || defined(unix) || defined(__unix) || defined(__linux__) || \
```

Comment:
Po co 3 macra unixowe i osobne linuxowe? Są jakieś platformy (które nas interesują) które definiują któreś z tych makr nie definiując pozostałych?

Reply:
Nie mam pojęcia, ale ostatnio jak nie było wszystkich to okazywało się, że się nie kompiluje. Po co by ich tyle tworzyli jeśli i tak każda byłaby używana? No więc założyłam, że bezpieczniej będzie dać wszystkie.